streams/write_all.ts

```diff
@@ -2,6 +2,7 @@
 
 /** Write all the content of the array buffer (`arr`) to the writer (`w`).
  *
+ * @example
  * ```ts
  * import { Buffer } from "https://deno.land/std@$STD_VERSION/io/buffer.ts";
  * import { writeAll } from "https://deno.land/std@$STD_VERSION/streams/write_all.ts";
```

Comment:
these are actually multiple examples in one; should be split up in idividual examples with titles